modules/caddyhttp/fileserver/caddyfile.go

```diff
@@ -107,6 +107,12 @@ func parseCaddyfile(h httpcaddyfile.Helper) (caddyhttp.MiddlewareHandler, error)
 				fsrv.Browse = new(Browse)
 				h.Args(&fsrv.Browse.TemplateFile)
 
+			case "reveal_symlink":
+				if fsrv.Browse == nil {
+					return nil, h.Err("reveal_symlink subdirectove requires browsing to be enabled")
+				}
+				fsrv.Browse.RevealSymlink = true
+
```

Comment:
I actually think this should be a subdirective of `browse` -- like within a block. Since it can't be used without it. Then it also has structure parity with the JSON config.

Reply:
@mholt What is exactly meant by within a block a here? Should it behave the same way as `[<template_file>]` does?

Reply:
So, for example:

```
file_server {
    browse {
        reveal_symlinks
    }
}
```

^ That's a block (for the `browse` subdirective)